tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -523,6 +509,11 @@ class TestBuildCommand_PythonFunctions_WithoutDocker(BuildIntegPythonBase):
     use_container = False
 
     def test_with_default_requirements(self):
+        # lucashuy: temporarily set docker check here
+        # until we refactor this set of tests
+        if IS_WINDOWS and not SKIP_DOCKER_TESTS and not runtime_supported_by_docker(self.runtime):
```

Comment:
I thought we are getting rid of `runtime_supported_by_docker` and using `pytest.mark` instead? Do we still need this function check here?

Reply:
This part of the code hasn't been touched yet , saving that for the next PR so that this one doesn't become too bloated in size. We'll need to keep this here for now.

Reply:
Ah, this particular check might be unecessary, let me remove it since this integ test isn't suppose to require an invoke anyways.